clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/ModifyAsgLaunchConfigurationOperation.groovy

```diff
@@ -78,6 +78,10 @@ class ModifyAsgLaunchConfigurationOperation implements AtomicOperation<Void> {
     props = props + description.properties.findResults { k, v -> (v != null && settingsKeys.contains(k)) ? [k, v] : null }.collectEntries()
     props.remove('class')
 
+    if (props.spotPrice == "") {
+      // Handle the case where you previously set a spot price and want to unset it
```

Comment:
comment might be a tad clearer as

`spotPrice of "" indicates that it should be removed regardless of value on source launch configuration`

?

Reply:
otherwise looks mergeable to me

Reply:
Definitely better wording. Changing then merging.